stats/datadog/collector.go

```diff
@@ -42,7 +43,7 @@ func (t tagHandler) processTags(tags map[string]string) []string {
 type Config struct {
 	common.Config
 
-	TagBlacklist lib.TagSet `json:"tagBlacklist,omitempty" envconfig:"TAG_BLACKLIST"`
+	TagBlacklist *stats.SystemTagSet `json:"tagBlacklist,omitempty" envconfig:"TAG_BLACKLIST"`
```

Comment:
I think @MStoykov is right in the discussion below (https://github.com/loadimpact/k6/pull/1148/files/6ec23ab5e93950d47cc1c6bcaebe31e1b126d570#r329612665), this is a regression, since it should be able to contain system (i.e. known to us at compile time) and non-system (i.e. user-defined, whatever they put in their script) tags. So it shouldn't use `stats.SystemTagSet`, rather it should use the map variant

Reply:
Ah right, I mis-understood this context